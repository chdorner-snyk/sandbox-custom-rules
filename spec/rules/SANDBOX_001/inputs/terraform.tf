resource "aws_s3_bucket" "invalid" {
    bucket = "test"
}
