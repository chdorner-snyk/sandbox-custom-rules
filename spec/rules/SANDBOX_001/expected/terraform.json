[
  {
    "passed": false,
    "ignored": false,
    "resource_id": "aws_s3_bucket.invalid",
    "resource_namespace": "spec/rules/SANDBOX_001/inputs/terraform.tf",
    "resource_type": "aws_s3_bucket",
    "severity": "high",
    "resources": [
      {
        "id": "aws_s3_bucket.invalid",
        "type": "aws_s3_bucket",
        "namespace": "spec/rules/SANDBOX_001/inputs/terraform.tf",
        "location": [
          {
            "filepath": "spec/rules/SANDBOX_001/inputs/terraform.tf",
            "line": 1,
            "column": 1
          }
        ]
      }
    ]
  }
]